satellite/repair/checker/checker.go

```diff
@@ -56,11 +54,11 @@ type Checker struct {
 	logger          *zap.Logger
 	Loop            sync2.Cycle
 	IrreparableLoop sync2.Cycle
-	monStats        durabilityStats
+	metaLoop        *metainfo.Loop
 }
 
 // NewChecker creates a new instance of checker
-func NewChecker(metainfo *metainfo.Service, repairQueue queue.RepairQueue, overlay *overlay.Cache, irrdb irreparable.DB, limit int, logger *zap.Logger, config Config) *Checker {
+func NewChecker(metainfo *metainfo.Service, repairQueue queue.RepairQueue, overlay *overlay.Cache, irrdb irreparable.DB, limit int, metaLoop *metainfo.Loop, logger *zap.Logger, config Config) *Checker {
```

Comment:
Because you have changed the signature of this function is a good chance to align it with our _argument order_ conventions https://github.com/storj/docs/blob/master/code/Style.md#argument-order